Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38218&attempt_id=816a656f-3d36-4784-9415-130893009977)

Attempted: monero, monero-gui

<details><summary>Partial log (click to expand)</summary><p>

```
[ 93%] Linking CXX executable ../../bin/monerod
clang-5.0: warning: argument unused during compilation: '-pie' [-Wunused-command-line-argument]
ld: framework not found PCSC
clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [src/daemon/CMakeFiles/daemon.dir/build.make:285: bin/monerod] Error 1
make[1]: *** [CMakeFiles/Makefile2:2892: src/daemon/CMakeFiles/daemon.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
builder for '/nix/store/kfh8znhdz79xka2h262128i5dmhwxwqq-monero-0.12.0.0.drv' failed with exit code 2
cannot build derivation '/nix/store/zxm9fx8zfbmmgh6xlwnna92my7lkm54p-monero-gui-2018-03-30.drv': 2 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/kfh8znhdz79xka2h262128i5dmhwxwqq-monero-0.12.0.0.drv', '/nix/store/zxm9fx8zfbmmgh6xlwnna92my7lkm54p-monero-gui-2018-03-30.drv' failed
```
</p></details>

--- a/pkgs/applications/altcoins/monero-gui/default.nix
+++ b/pkgs/applications/altcoins/monero-gui/default.nix
@@ -2,32 +2,34 @@
 , makeWrapper, makeDesktopItem
 , qtbase, qmake, qtmultimedia, qttools
 , qtgraphicaleffects, qtdeclarative
-, qtlocation, qtquickcontrols, qtwebchannel
+, qtlocation, qtquickcontrols2, qtwebchannel
 , qtwebengine, qtx11extras, qtxmlpatterns
 , monero, unbound, readline, boost, libunwind
+, pcsclite, zeromq, cppzmq, pkgconfig
 }:
 
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "monero-gui-${version}";
-  version = "0.11.1.0";
+  version = "2018-03-31";
 
   src = fetchFromGitHub {
     owner  = "monero-project";
     repo   = "monero-gui";
-    rev    = "v${version}";
-    sha256 = "01d7apwrv8j8bh7plvvhlnll3ransaha3n6rx19nkgvfn319hswq";
+    rev    = "fbe5ba831795008361f4de4347e7ecb6d868b4eb";
+    sha256 = "06cncwk4mxfw1rqwlwisasvangl73xyqwj4g6r9j85j5x4xy0k5s";
   };
 
-  nativeBuildInputs = [ qmake ];
+  nativeBuildInputs = [ qmake pkgconfig ];
 
   buildInputs = [
     qtbase qtmultimedia qtgraphicaleffects
-    qtdeclarative qtlocation qtquickcontrols
+    qtdeclarative qtlocation qtquickcontrols2
     qtwebchannel qtwebengine qtx11extras
     qtxmlpatterns monero unbound readline
-    boost libunwind makeWrapper
+    boost libunwind pcsclite zeromq cppzmq
+    makeWrapper
   ];
 
   patches = [

--- a/pkgs/applications/altcoins/monero-gui/move-log-file.patch
+++ b/pkgs/applications/altcoins/monero-gui/move-log-file.patch
@@ -1,8 +1,8 @@
 diff --git a/main.cpp b/main.cpp
-index 1a9a979..2316929 100644
+index c03b160..a8ea263 100644
 --- a/main.cpp
 +++ b/main.cpp
-@@ -74,10 +74,6 @@ int main(int argc, char *argv[])
+@@ -80,14 +80,16 @@ int main(int argc, char *argv[])
  //    qDebug() << "High DPI auto scaling - enabled";
  //#endif
  
@@ -13,9 +13,6 @@ index 1a9a979..2316929 100644
      MainApp app(argc, argv);
  
      qDebug() << "app startd";
-@@ -86,6 +82,13 @@ int main(int argc, char *argv[])
-     app.setOrganizationDomain("getmonero.org");
-     app.setOrganizationName("monero-project");
  
 +    // Log settings
 +    QString logfile =
@@ -23,20 +20,19 @@ index 1a9a979..2316929 100644
 +      + "/monero-wallet-gui.log";
 +    Monero::Wallet::init(argv[0], logfile.toUtf8().constData());
 +
-+
-     filter *eventFilter = new filter;
-     app.installEventFilter(eventFilter);
- 
+     app.setApplicationName("monero-core");
+     app.setOrganizationDomain("getmonero.org");
+     app.setOrganizationName("monero-project");
 diff --git a/src/libwalletqt/Wallet.cpp b/src/libwalletqt/Wallet.cpp
-index 8525bf3..6967b24 100644
+index 74649ce..fe1efc6 100644
 --- a/src/libwalletqt/Wallet.cpp
 +++ b/src/libwalletqt/Wallet.cpp
-@@ -613,7 +613,7 @@ QString Wallet::getDaemonLogPath() const
- 
- QString Wallet::getWalletLogPath() const
- {
--    return QCoreApplication::applicationDirPath() + "/monero-wallet-gui.log";
-+    return QStandardPaths::writableLocation(QStandardPaths::CacheLocation) + "/monero-wallet-gui.log";
+@@ -729,7 +729,7 @@ QString Wallet::getWalletLogPath() const
+ #ifdef Q_OS_MACOS
+     return QStandardPaths::standardLocations(QStandardPaths::HomeLocation).at(0) + "/Library/Logs/" + filename;
+ #else
+-    return QCoreApplication::applicationDirPath() + "/" + filename;
++    return QStandardPaths::writableLocation(QStandardPaths::CacheLocation) + filename;
+ #endif
  }
  
- Wallet::Wallet(Monero::Wallet *w, QObject *parent)

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -16722,7 +16722,9 @@ with pkgs;
     boost = boost15x;
   };
 
-  monero-gui = libsForQt5.callPackage ../applications/altcoins/monero-gui { };
+  monero-gui = libsForQt5.callPackage ../applications/altcoins/monero-gui {
+    boost = boost15x;
+  };
 
   xmr-stak = callPackage ../applications/misc/xmr-stak {
     hwloc = hwloc-nox;

